src/infrastructure/organisations/api.js

```diff
@@ -29,12 +29,14 @@ const callOrganisationsApi = async (endpoint, method, body, correlationId) => {
   }
 };
 
-const listServiceUsers = async (serviceId, page, pageSize, correlationId) => {
+const listServiceUsers = async (serviceId, userIds, page, pageSize, correlationId) => {
   const token = await jwtStrategy(config.applications.service).getBearerToken();
   try {
+    const url = `${config.organisations.service.url}/services/${serviceId}/users?page=${page}&pageSize=${pageSize}`;
```

Comment:
I think we should not send params in the URL for a POST, send them all in the body (page, pageSize, userIds), then in the endpoint you handle that it can be a get or a post so you get the stuff from req.body or req.params